lib/debugger.coffee

```diff
@@ -31,11 +35,15 @@ class ProcessManager extends EventEmitter
         appArgs = @atom.config.get('node-debugger.appArgs')
         port = @atom.config.get('node-debugger.debugPort')
         env = @parseEnv @atom.config.get('node-debugger.env')
+        defaultFile = @atom.project.getPaths()[0] + '/' + @atom.config.get('node-debugger.defaultFile')
```

Comment:
this is a first draft line of code so to speak.
`project.getPaths()` returns an array of paths.  But i haven't messed with Atom enough to know when you'd end up with more than one path.  And if so, which one do you use?  Can you assume the first one is the project root?

the reason for this is to be able to get the fully qualified file path for the debugger-  So that it knows where the `defaultFile` is.  I'm totally open to other ideas about how to better do this.  Compared to blindly picking the first item in an array.  I just haven't gotten through enough of the docs to have a better idea.